hack/boilerplate/boilerplate.generatego.txt

```diff
@@ -0,0 +1,3 @@
+// Copyright (c) Mattermost, Inc. All Rights Reserved.
```

Comment:
Missing date range

Reply:
the pattern is left without a date when is generated files

Reply:
i'm going to change that to add that date as well

Reply:
done, thanks for your review @gigawhitlocks